### DISCUSSION TITLE
API Token and Port Information

### DISCUSSION THREAD

**Discussion by ghost** [gracingpro/blauberg-homeassistant#4] [Q&A] [answered]
Hi,
Could you please guide me on where I can find the necessary API Token and Port information for the Blauberg config?

**Comment by JenertsA** [comment, marked as answer]
Hi, had the same question. 
Turns out that this is not a setting you gather from Blauberg. This is port you expose for UI interaction and the API token is key needed to authenticate there. 

At least that's what I figured.
